charts/konk/templates/configmap.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "konk.fullname" . }}-configmap
+  labels:
+    app: {{ include "konk.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+  start.sh: |
+    kubeadm init phase certs all
+    kubeadm init phase kubeconfig admin
+    VAL=$(kubectl -n {{ .Release.Namespace }} get secret {{ include "konk.fullname" . }}-etcd-cert --ignore-not-found)
+    if [[ $VAL == "" ]]
+    then
+      kubectl -n {{ .Release.Namespace }} create secret generic {{ include "konk.fullname" . }}-etcd-cert --from-file=/etc/kubernetes/pki/etcd/ca.crt --from-file=/etc/kubernetes/pki/etcd/server.crt --from-file=/etc/kubernetes/pki/etcd/server.key --from-file=/etc/kubernetes/pki/apiserver-etcd-client.crt --from-file=/etc/kubernetes/pki/apiserver-etcd-client.key
```

Comment:
It would be good to put these into two different secrets, one for the apiserver container and one for the etcd container, for the concept of "least privilege". The containers don't need access to each others private keys.

Reply:
Split the certs into 2 separate secrets for apiserver and etcd.